src/simulation/dynamics/spinningBodies/spinningBodiesOneDOF/_UnitTest/test_spinningBodyOneDOFStateEffector.py

```diff
@@ -262,4 +275,4 @@ def spinningBody(show_plots):
 
 
 if __name__ == "__main__":
-    spinningBody(True)
+    spinningBodyTest(True, 0.0, False)
```

Comment:
update this to be `spinningBody` with three arguments.

Reply:
Done.